analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LockReleasedAllPaths.ReaderWriterLockSlim.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Threading;
+
+class Program
+{
+    private ReaderWriterLockSlim readerWriterLockSlim = new ReaderWriterLockSlim();
+
+    public void Method1(bool condition)
+    {
```

Comment:
Please add variant with `EnterReadLock()` and `ExitWriteLock()`. Linear and conditional to demonsrate that we don't care about the kind of lock